website/docs/r/nsxt_edgegateway.html.markdown

```diff
@@ -374,15 +419,31 @@ added capability of automatically allocating a specific number of IPs from the b
   defined in any of the `subnet_with_ip_count` blocks
 * `allocated_ip_count` (Required) - Number of allocated IPs from that particular subnet
 
+<a id="edgegateway-subnet-external-network"></a>
+
+### NSX-T Segment backed network attachment using `external_network` block
+
+This option can attach NSX-T Segment backed External Networks
+
+* `external_network_id` - (Required) - ID of NSX-T Segment backed External Network
+* `gateway` - (Required) - Gateway for a subnet in external network
```

Comment:
Please chose one of "External Network" or "external network" in this bullet list for consistency :)

Reply:
It was 9 occurrences for lower cases vs. 8 for upper cases in this document. I made them all lower cased now :)